coati/generator.py

```diff
@@ -73,32 +73,41 @@ def generate(project_name, ppt_path):
     spaces = " " * 12
     slide_tuples = '['
 
-    path_builders = os.path.join(path, 'builders/')
+    #Generate builders/ folder prior slides creation
+    _generate_path(os.path.join(path, 'builders/'))
+    log.warn('create folder %s', "./builders/")
```

Comment:
I think the log message should be based on the thing that is doing rather than the visual representation